src/plugins/data/server/index.ts

```diff
@@ -56,6 +55,11 @@ export {
   // utils
   parseInterval,
   isNestedField,
+  // fieldFormats
+  fieldFormats,
+  FieldFormatsGetConfigFn,
+  FieldFormatConfig,
+  FieldFormatsRegistry,
```

Comment:
Currently (in both `public/index` and `server/index`) we have static exports separated from type exports. To avoid confusion maybe we should stick the namespace with the static exports for the time being?

Once we move everything over to this structure, it should be more intuitive whether something is static or a type, but in the interim it may help make things clearer.

Reply:
This transition shouldn't take more than a few days, so I don't think it's worth leaving it for later.
What do you think?

Reply:
If we're planning to make the transition right away, and you think it's only a couple days, then I have no concerns about keeping it as-is. I only brought it up in the event that the data plugin might sit like this for awhile before we come back to clean up the rest.